### PR TITLE
Add Exercise 7

### PR DESCRIPTION
**Pull request by nguyenvanman**

### CODE REVIEW

File: Exercise_7/Program.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            item1 = item2;
+            item2 = temp;
+        }
+        static void Sort<T>(T[] list)
+        {
+            for (var i = 0; i < list.Length - 1; i++)
+            {
+                for (var j = i + 1; j < list.Length; j++)
+                {
+                    if (((ICompare) list[i]).IsBigger((ICompare) list[j]))
```
Comment (voquanghoa) [inline Exercise_7/Program.cs]: You should use [Constraints](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/generics/constraints-on-type-parameters) or [this](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/where-generic-type-constraint) instead.

File: Exercise_7/Program.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                        Swap(ref list[i],ref list[j]);
+                    }
+                }
+            }
+        }
+        static void ShowList<T>(T[] list)
+        {
+            for (var i = 0; i < list.Length; i++)
+            {
+                Console.WriteLine(list[i].ToString());
```
Comment (voquanghoa) [inline Exercise_7/Program.cs]: Try `list.ForEach(Console.WriteLine)`

File: Exercise_7/Student.cs [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        }
+
+        bool ICompare.IsBigger(ICompare another)
+        {
+            return Score > ((Student) another).Score;
+        }
+
+        public override string ToString()
+        {
+            return "Name: " + Name + "\tBirthday: "
```
Comment (voquanghoa) [inline Exercise_7/Student.cs]: Use `string interpolation`

**Review by voquanghoa — CHANGES_REQUESTED** [review]

File: Exercise_7/Program.cs (hunk 1 above)
Comment (nguyenvanman) [reply Exercise_7/Program.cs]: Done

File: Exercise_7/Program.cs (hunk 2 above)
Comment (nguyenvanman) [reply Exercise_7/Program.cs]: Done

File: Exercise_7/Student.cs (hunk 3 above)
Comment (nguyenvanman) [reply Exercise_7/Student.cs]: Done

**Review by voquanghoa — APPROVED** [review]

**Pull request merged**